app/bundles/DynamicContentBundle/Form/Type/DynamicContentType.php

```diff
@@ -56,6 +103,19 @@ public function buildForm(FormBuilderInterface $builder, array $options)
             ]
         );
 
+        $builder->add(
+            'slotName',
+            'text',
+            [
+                'label'      => 'mautic.dynamicContent.send.slot_name',
```

Comment:
We need a callback constraint on this to validate not blank if isCampaignBased = false

Reply:
We also need a callback constraint on filters to invalidate the field if filters is empty when isCampaignBased = false

Reply:
Added on latest commit